app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,28 @@
+class VotesController < ApplicationController
+
+  def index
+      @votes = Vote.all
+  end
+
+  def show
+    @vote = Vote.find_by(id: params[:id])
+    if @vote.nil?
+      head :not_found
+    end
```

Comment:
Hm, which requirement/user story were you working on when trying to view a specific vote?